l5kit/l5kit/simulation/unroll.py

```diff
@@ -39,6 +63,26 @@ def get_scene_id(self) -> int:
         """
         return self.scene_id
 
+    @staticmethod
+    def build_trajectory_states(frames: np.ndarray) -> torch.Tensor:
+        """
+        Convert frames into a torch trajectory
+        :param frames: the scene frames
+        :return: the trajectory
+        """
+        trajectory_states = torch.zeros(len(frames), len(TrajectoryStateIndices.__members__), dtype=torch.float)
```

Comment:
```suggestion
        trajectory_states = torch.zeros(len(frames), len(TrajectoryStateIndices), dtype=torch.float)
```

Reply:
my IDE was complaining about this, but it works indeed :)